src/renderer/helpers/playlists.js

```diff
@@ -45,6 +49,10 @@ function compareTwoPlaylistItems(a, b, sortOrder, collator) {
       return collator.compare(a.author, b.author)
     case SORT_BY_VALUES.AuthorDescending:
       return collator.compare(b.author, a.author)
+    case SORT_BY_VALUES.VideoLengthAscending:
+      return a.lengthSeconds - b.lengthSeconds
+    case SORT_BY_VALUES.VideoLengthDescending:
+      return b.lengthSeconds - a.lengthSeconds
```

Comment:
> it will be sorted as if their length is zero

Is this code doing what's claimed?

Reply:
Are you sure? 
I saw `if (!Object.hasOwn(v, 'lengthSeconds') || isNaN(v.lengthSeconds))` in `checkPlaylist` so either that's wrong or `lengthSeconds` can be `NaN`/`undefined`
I haven't tested this myself yet to be done tomorrow

Reply:
I think this is running into javascript truthy behaviour, I'll do some more tests

Reply:
It can be left `undefined` if not evaluated correctly, but I think I should also check if it's zero

Reply:
I've come to the conclusion tthat the video object will always have `.lengthSeconds`, but depending on how it's evaluated, it could spit out an undefined, so I'll keep that in the code. I have also included the zero-check behaviour.

Reply:
If I add shorts to a playlist, they have string values
![image](https://github.com/user-attachments/assets/30783141-8e94-4c5b-a2c3-6ff927bef89f)

I think it's best to use the same check as the one below and use `0` if check failed (See another line comment)

Reply:
I think that may be a bug in whatever system parses shorts, but we can put in a check here too.

Reply:
> I think that may be a bug in whatever system parses shorts, but we can put in a check here too.

Nope, YouTube doesn't return durations for shorts on the shorts channel tab or in playlists that only contain shorts, so we set it to an empty string, the alternative would be `null` but the empty string fit best into the existing logic at the time.

Reply:
That's fair enough, though I still feel a `null` would be better than an empty string for a integer value, but all in all, this code won't care.